konnectd/ui/build/static/css/main.5634be8f.chunk.css

```diff
@@ -0,0 +1,2 @@
rl(../../static/media/roboto-latin-300italic.d69924b9.woff2) format("woff2"),url(../../static/media/roboto-latin-300italic.210a7c78.woff) format("woff")}@font-face{font-family:"Roboto";font-style:normal;font-display:swap;font-weight:400;src:local("Roboto Regular "),local("Roboto-Regular"),url(../../static/media/roboto-latin-400.5d4aeb4e.woff2) format("woff2"),url(../../static/media/roboto-latin-400.bafb105b.woff) format("woff")}@font-face{font-family:"Roboto";font-style:italic;font-display:swap;font-weight:400;src:local("Roboto Regular italic"),local("Roboto-Regularitalic"),url(../../static/media/roboto-latin-400italic.d8bcbe72.woff2) format("woff2"),url(../../static/media/roboto-latin-400italic.9680d5a0.woff) format("woff")}@font-face{font-family:"Roboto";font-style:normal;font-display:swap;font-weight:500;src:local("Roboto Medium "),local("Roboto-Medium"),url(../../static/media/roboto-latin-500.28546717.woff2) format("woff2"),url(../../static/media/roboto-latin-500.de8b7431.woff) format("woff")}@font-face{font-family:"Roboto";font-style:italic;font-display:swap;font-weight:500;src:local("Roboto Medium italic"),local("Roboto-Mediumitalic"),url(../../static/media/roboto-latin-500italic.510dec37.woff2) format("woff2"),url(../../static/media/roboto-latin-500italic.ffcc050b.woff) format("woff")}@font-face{font-family:"Roboto";font-style:normal;font-display:swap;font-weight:700;src:local("Roboto Bold "),local("Roboto-Bold"),url(../../static/media/roboto-latin-700.037d8304.woff2) format("woff2"),url(../../static/media/roboto-latin-700.cf6613d1.woff) format("woff")}@font-face{font-family:"Roboto";font-style:italic;font-display:swap;font-weight:700;src:local("Roboto Bold italic"),local("Roboto-Bolditalic"),url(../../static/media/roboto-latin-700italic.010c1aee.woff2) format("woff2"),url(../../static/media/roboto-latin-700italic.846d1890.woff) format("woff")}@font-face{font-family:"Roboto";font-style:normal;font-display:swap;font-weight:900;src:local("Roboto Black "),local("Roboto-Black"),url(../../static/media/roboto-latin-900.19b7a0ad.woff2) format("woff2"),url(../../static/media/roboto-latin-900.8c2ade50.woff) format("woff")}@font-face{font-family:"Roboto";font-style:italic;font-display:swap;font-weight:900;src:local("Roboto Black italic"),local("Roboto-Blackitalic"),url(../../static/media/roboto-latin-900italic.7b770d6c.woff2) format("woff2"),url(../../static/media/roboto-latin-900italic.bc833e72.woff) format("woff")}#bg{background-color:#fff}#bg>div{position:absolute;top:0;bottom:0;left:0;right:0;background-size:cover;background-repeat:no-repeat;background-position:50%;z-index:0}#loader{color:#fff;text-shadow:#000 0 0 1px}#bg-enhanced{opacity:0;-webkit-transition:opacity 1s;transition:opacity 1s}#bg-enhanced.enhanced{background-image:url(../../static/media/background.d469fc8c.jpg);opacity:1}#bg-enhanced.enhanced:after{content:"";position:absolute;top:0;bottom:0;left:0;right:0;background-size:cover;background-repeat:no-repeat;background-position:50%}
```

Comment:
Codacy found an issue: [Expected single space before "}" of a single-line block (block-closing-brace-space-before)](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=6337760)